packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -16,22 +16,24 @@ export interface TextProps {
   color?: keyof GrafanaTheme2['colors']['text'] | 'error' | 'success' | 'warning' | 'info';
   /** Use to cut the text off with ellipsis if there isn't space to show all of it. On hover shows the rest of the text */
   truncate?: boolean;
+  /** If true, show the text as italic. False by default */
+  italic?: boolean;
   /** Whether to align the text to left, center or right */
   textAlignment?: CSSProperties['textAlign'];
   children: React.ReactNode;
 }
 
 export const Text = React.forwardRef<HTMLElement, TextProps>(
-  ({ as, variant, weight, color, truncate, textAlignment, children }, ref) => {
+  ({ element = 'span', variant, weight, color, truncate, italic = false, textAlignment, children }, ref) => {
     const styles = useStyles2(
       useCallback(
-        (theme) => getTextStyles(theme, variant, color, weight, truncate, textAlignment),
-        [color, textAlignment, truncate, weight, variant]
+        (theme) => getTextStyles(theme, variant, color, weight, truncate, italic, textAlignment),
+        [color, textAlignment, truncate, italic, weight, variant]
       )
     );
 
     return createElement(
-      as,
+      element,
```

Comment:
Do we need to readd here somewhere what is the default variant for each element? Feel like we have lost that in the process and it is now just going back to the default css?